### PR TITLE
Laravel 58

### PR DESCRIPTION
**Pull request by dejwCake**

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-Route::middleware(['web', 'auth:' . config('admin-auth.defaults.guard'), 'admin'])->group(function () {
```
Comment (palypster) [inline routes/web.php]: @dejwCake should  `web` be also removed?

File: routes/web.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+Route::middleware(['auth:' . config('admin-auth.defaults.guard')])->group(function () {
```
Comment (SamuelT-ST) [inline routes/web.php]: Now when `brackets/admin-auth` is not required, will `admin-auth.defaults.guard` work?

File: routes/web.php (hunk 1 above)
Comment (dejwCake) [reply routes/web.php]: Both middlewares has been moved to Service provider.

File: routes/web.php (hunk 2 above)
Comment (dejwCake) [reply routes/web.php]: Yes, it should work, because "dummy" config values are registered in service provider.

**Comment by dejwCake** [comment]
This is a blocker for laravel 58, so please review.

**Pull request merged**